framework/doc/content/syntax/BCs/index.md

```diff
@@ -1,29 +1,49 @@
 # BCs System
 
-!alert! construction title=Undocumented System
-The BCs system has not been documented. The content listed below should be used as a starting
-point for documenting the system, which includes the typical automatic documentation associated with
-a system; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
+The `BCs` system is used to impose boundary conditions in a finite element problem. Each `BC` object
+sets the boundary condition for a single variable/equation, though multiple boundaries may be specified at once.
 
-```markdown
-# BCs System
+!alert note
+See [FVBCs](syntax/FVBCs/index.md) for boundary conditions of finite volume problems.
 
-## Overview
+Some of the main types of boundary conditions are:
+- [Dirichlet Boundary conditions](DirichletBC.md) to set the value of a variable on a boundary
+- [Neumann Boundary conditions](NeumannBC.md) to set a condition on the derivative of a variable. Depending on the
```

Comment:
Strictly speaking a Neumann BC is a condition on the flux of a variable; that's only equal to the derivative for simple problems like the Laplacian.  And looking at `NeumannBC.C`, it looks to me like it's applying a forcing that would correspond to the flux, not the derivative, in cases where a constitutive term makes those differ.

Reply:
I was thinking that too but wikipedia disagrees
https://en.wikipedia.org/wiki/Neumann_boundary_condition

Reply:
I ll reword to make that clear

Reply:
> always

Now I'm kind of wondering about exceptions; the theorems I recall always started with something like "let c>0 = min_Ω (k)", but I'm not sure whether disallowing any singularity in k was a *necessary* postulate or just a *simple* one...

Reply:
I think the controversy for me is more around fluxes that are not derivatives, even with a (reasonable) coefficient.
The wording right now should be safe?

Reply:
Looks reasonable now.